docker/conf/nginx.conf

```diff
@@ -60,43 +60,6 @@ http {
     server_name         localhost;
     access_log          /dev/null main;
 
-    location ~ (/app/kibana|/app/sense|/bundles/|/kibana4|/status|/plugins|/elasticsearch|/api/sense/proxy|/kibana|/kibana/goto) {
-      proxy_http_version   1.1;
-      proxy_pass           http://127.0.0.1:5601;
-      proxy_read_timeout   90;
-      proxy_set_header     Upgrade             $http_upgrade;
-      proxy_set_header     Connection 'upgrade';
-      proxy_set_header     Host                $http_host;
-      proxy_set_header     X-Real-IP           $remote_addr;
-      proxy_set_header     X-Forwarded-For     $proxy_add_x_forwarded_for;
-      proxy_set_header     X-Forwarded-Proto   $scheme;
-      proxy_cache_bypass   $http_upgrade;
-      rewrite              /kibana/(.*)$ /$1 break;
-    }
-    location ~ ^/_aliases$ {
-      proxy_pass http://127.0.0.1:9200;
-      proxy_read_timeout 90;
-    }
-    location ~ ^/_nodes$ {
-      proxy_pass http://127.0.0.1:9200;
-      proxy_read_timeout 90;
-    }
-    location ~ ^/.*/_search$ {
-      proxy_pass http://127.0.0.1:9200;
-      proxy_read_timeout 90;
-    }
-    location ~ ^/.*/_mapping$ {
-      proxy_pass http://127.0.0.1:9200;
-      proxy_read_timeout 90;
-    }
-    # Password protected end points
-    location ~ ^/.kibana/dashboard/.*$ {
-      proxy_pass http://127.0.0.1:9200;
-      proxy_read_timeout 90;
-    }
-    location ~ ^/.kibana/temp.*$ {
-      proxy_pass http://127.0.0.1:9200;
-      proxy_read_timeout 90;
-    }
+    proxy_pass http://127.0.0.1:5601;
```

Comment:
curious why this line is here, and not in  docker/compose/nginx/files/nginx.conf - just seems a bit duplicative?

Reply:
This file is specific for the docker container where everything runs in one container, the file you pointed to is used for docker with multiple containers.

Reply:
ahh ok! I missed that. Thanks!